Automattic/wp-calypso
# Editing Toolkit: Update Newspack Blocks to v1.19.0

#### Changes proposed in this Pull Request

* Editing Toolkit: Update Newspack Blocks to 1.19.0.

This new version of Newspack Blocks includes a much-requested fix for a "Load More" bug. Details on the bug and its resolution are [here](https://github.com/Automattic/newspack-blocks/issues/404) and [here](https://github.com/Automattic/newspack-blocks/pull/672).

Note that v1.18+ of Newspack Blocks also uses a new custom REST endpoint to fetch posts in the editor. For this reason, we needed to add a bit of code to `apps/editing-toolkit/editing-toolkit-plugin/newspack-blocks/index.php` to localize the endpoint URL so that it first gets run through the FSE filter that rewrites the `newspack-blocks/v1` REST namespace to `wpcom/v2`.

#### Testing instructions



* Sync this branch and build it to a sandbox. Make sure to sandbox the public API as well, or the new custom REST route won't work.
* Follow the testing instructions in https://github.com/Automattic/newspack-blocks/pull/672. 

Fixes https://github.com/Automattic/newspack-blocks/issues/404.

Closes https://github.com/Automattic/wp-calypso/issues/48253
